chainbridge/README.md

```diff
@@ -0,0 +1 @@
+License: Unlicense
```

Comment:
We may want to use `GNU Lesser General Public License v3.0`

This is the same that the repo was originally licensed under